.sonarlint/sonarlint-visualstudio/CSharp/SonarLint.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The file is different from the previous SonarLint.xml, notably the license tag is not there anymore. However, I think that's ok, since I develop without our extension installed and thus don't see it anyway...